manifoldco/ui
# Implement resizing of configurable features

## Reason for change




## Testing


- Create a configured plan in storybook
- go to resize plan selector and verify that all values are the same as when the resource was created.
- change the configured values and click the resize button
- check in resource details that the values have been updated on the resource

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
